server/src/test/java/org/elasticsearch/index/mapper/KeywordFieldTypeTests.java

```diff
@@ -55,28 +52,6 @@
 
 public class KeywordFieldTypeTests extends FieldTypeTestCase<KeywordFieldType> {
 
-    @Before
-    public void addModifiers() {
-        addModifier(t -> {
```

Comment:
I think we used to be able to change these.

Reply:
yeah, these Modifier tests are basically useless now